scalr/utils/file_utils.py

```diff
@@ -172,31 +182,33 @@ def load_test_data_from_config(data_config):
     Args:
         data_config: Data config.
     """
-    flow_logger = FlowLogger('File Utils')
-
     train_val_test_paths = data_config.get('train_val_test')
-    if not train_val_test_paths:
-        raise ValueError('Split Datapaths not given')
+    datapath = _get_datapath_from_config(data_config)
 
-    if train_val_test_paths.get('feature_subset_datapaths'):
-        test_data = read_data(
-            path.join(train_val_test_paths['feature_subset_datapaths'], 'test'))
-        flow_logger.info('Test Data Loaded from Feature subset datapaths')
+    test_data = read_data(path.join(train_val_test_paths[datapath], 'test'))
```

Comment:
okay, so here the priority is 1. feature subsets 2. final data 3. data splits... final should come before feature right?

Reply:
final implies final data after data-ingestion step
final: pre-processes data. After norms, etc.